R/utils-custom.R

```diff
@@ -12,22 +12,33 @@
 theme_tiltplot <- function() {
   theme_classic() +
     theme(
-      plot.title = element_text(hjust = 0.5, size = 16),
-      axis.title = element_text(size = 12),
-      axis.text = element_text(size = 10),
-      legend.title = element_text(size = 12),
-      legend.text = element_text(size = 10),
+      plot.title = element_text(hjust = 0.5, size = 16, family = tilt_headline_font()),
+      strip.text = element_text(family = tilt_text_font()),
+      axis.title = element_text(size = 10, family = tilt_text_font()),
+      axis.text = element_text(size = 10, family = tilt_text_font()),
+      legend.title = element_text(size = 12, family = tilt_text_font()),
+      legend.text = element_text(size = 10, family = tilt_text_font()),
       legend.position = "bottom",
       panel.grid.major.y = element_blank(),
       panel.grid.minor.y = element_blank(),
       panel.spacing.y = unit(0.5, "lines")
     )
 }
 
-score_colors <- function(...) c("low" = "#007F00", "medium" = "#FFC300", "high" = "#FF5733")
+tilt_text_font <- function() "Roboto"
+
+tilt_headline_font <- function() "Roboto Condensed"
+
+score_colors <- function(...) c("low" = "#b3d15d", "medium" = "#f6cb4e", "high" = "#e3693b")
 
 fill_score_colors <- function() scale_fill_manual(values = score_colors())
 
+hex_to_rgb <- function(hex) {
+  rgb <- col2rgb(hex)
```

Comment:
Do you need `usethis::use_import_from("grDevices", "col2rgb")`?

In a failing check I see

```
hex_to_rgb: no visible global function definition for ‘col2rgb’
  (/home/runner/work/tiltPlot/tiltPlot/check/tiltPlot.Rcheck/00_pkg_src/tiltPlot/R/utils-custom.R:37)
Undefined global functions or variables:
  col2rgb
Consider adding
  importFrom("grDevices", "col2rgb")
```